added report-disk command (#4444)
influxdata/docs-v2: content/enterprise_influxdb/v1.10/tools/influx_inspect.md

* [`dumptsmwal`](#dumptsmwal): Dump all data from a WAL file.  
* [`export`](#export): Exports raw data from a shard in InfluxDB line protocol format.
* [`report`](#report): Displays a shard level report.

> * [`report-disk`](#report-disk): Reports disk usage by shards and measurements.  

Reviewer: ```suggestion
* [`report-disk`](#report-disk): Reports disk usage by shard and measurement.  
```

Default value is `false`.
Note: This can use a lot of memory.

### `report-disk`

> Use the `report-disk` command to review disk usage by shards and measurements for TSM files in a specified directory. Useful for determining disk usage for capacity planning and identifying which measurement or shard is using the most space. The default location is `~/.influxdb/data/`.

Reviewer: ```suggestion
Use the `report-disk` command to review TSM file disk usage per shard and measurement in a specified directory. Useful for capacity planning and identifying which measurement or shard is using the most disk space. The default directory path `~/.influxdb/data/`.
```

Calculates the total disk size by database (`db`), retention policy (`rp`), shard (`shard`), tsm file (`tsm_file`), and measurement (`measurement`).

#### Syntax

```
influx_inspect report-disk [ options ] <data_dir>
```

#### Options

Optional arguments are in brackets.

##### [ `-detailed` ]

> Include this flag to report disk usage by measurement.

Reviewer: ```suggestion
Report disk usage by measurement.
```
